### PR TITLE
Fix Docker setup

### LINKED ISSUE #4713: Docker installation hangs running on Windows
Following the [DOCKER.md](https://github.com/gravitystorm/openstreetmap-carto/blob/master/DOCKER.md) readme, on the step `docker-compose up kosmtik` it seems that one package downloaded from `apt` expects an answer via keyboard, which doesn't seem to be possible at least running it on Windows with PowerShell.

image

The gray output seems to be the CLI output, I can see that `apt` updates, but eventually when it start to install the packages, it seems to hang on this output. I've tried entering a number but it doesn't seem to accept.

I also tried running it from inside WSL, but after installing `docker-compose`, it fails to run the import script:

image

### PR DESCRIPTION
**Pull request by SlowMo24**
Fixes #4713 #4683 

Changes proposed in this pull request:
- update Dockerfile to fix build issues
- update documentation to reference `127.0.0.1` instead of `localhost`

We should probably update the complete Docker stack and also help https://github.com/kosmtik/kosmtik to update but this PR at least makes the current code usable again.

~Test rendering with links to the example places:~

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 FROM ubuntu:focal
 
+# https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
```
Comment (pnorman) [inline Dockerfile:5]: I'd rather this was the system time zone or UTC, rather than a specific local timezone.

**Review by pnorman — CHANGES_REQUESTED** [review]
Small tz change.

A [Serverfault answer](https://serverfault.com/a/1044731/297026) says that TZ and DEBIAN_FRONTEND doesn't work on Ubuntu 20.04, but if someone has tested this, I'm fine with that.

File: Dockerfile (hunk 1 above)
Comment (Firefishy) [reply Dockerfile:5]: Then ENV TZ line isn't need as UTC is used by default.

Comment (SlowMo24) [reply Dockerfile]: thanks for the review. I have removed the ENV TZ and the build works fine

**Review by pnorman — APPROVED** [review]

**Pull request merged**